### PR TITLE
Stray OAuth Requirement

### PR DESCRIPTION
**Pull request by andyjohnson**
Fixed a MUST in normative language - moved into a requirement.

### CODE REVIEW

**Comment by andyjohnson** [comment]
Fixes #965

**Comment by ljwolford** [comment]
+1

**Comment by garemoko** [comment]
Since there's only 1 requirement here, should we just have 1 list of Requirements for OAuth here: https://github.com/adlnet/xAPI-Spec/blob/ForStrayMust/xAPI-Communication.md#requirements-18 ? That was what I thought you meant in the issue.

**Comment by andyjohnson** [comment]
I like that suggestion, moved in the latest commit

File: xAPI-Communication.md [hunk 1]
```diff
@@ -1860,8 +1860,7 @@ The following table lists xAPI scope values:
 </table>
 
 ###### <a name="4.2.s3"></a>OAuth Extended Parameters
```
Comment (garemoko) [inline xAPI-Communication.md]: @andyjohnson you also need to delete this heading as it now has no content.

**Comment by andyjohnson** [comment]
Lol, I guess that did leave the section empty.  I got rid of the description header but didn't realize that was all in that section!  Fixed.

**Comment by garemoko** [comment]
+1

**Comment by ljwolford** [comment]
+1

**Pull request merged**